### ISSUE TITLE
ConTroll/controll/lib/base.php - wrong path for vendor/autoload.php

### ISSUE THREAD

**Issue by PaulTimmins** [MichaelRafferty/ConTroll#38]
I had to hardcode
require_once("/home/capric/regtest.capricon.org/ConTroll/Composer/vendor/autoload.php");
to get the site to work.

I'm sure it should have a relative path like the other require_once ones but I was in a hurry.

**Comment by sydwhv** [comment]
no, that is not required and incorrect.  It tells me you didn't do something right in the setup.
